docs/source/deployment/tgi.rst

```diff
@@ -0,0 +1,106 @@
+TGI
+=====================
+
+Hugging Face's Text Generation Inference (TGI) is a production-ready framework specifically designed for deploying and serving large language models (LLMs) for text generation tasks. It offers a seamless deployment experience, powered by a robust set of features:
+
+* **Speculative Decoding:**  Accelerates generation speeds.
```

Comment:
If you advertise for those (speculative decoding and versatile device support, specifically), please at least provide a link to the documentation on how to use those.